nest-forms-backend/src/nases-consumer/subservices/email-forms.subservice.ts

```diff
@@ -81,7 +90,7 @@ export default class EmailFormsSubservice {
 
     if (
       formDefinition.type !== FormDefinitionType.Email ||
-      !formDefinition.email
+      !formDefinition.email.address
```

Comment:
If this is enforced by TypeScript I would skip this check altogether.

Reply:
If you want, you might add check to https://github.com/bratislava/konto.bratislava.sk/blob/master/forms-shared/tests/definitions/formDefinitions.ts e.g. to validate if the email is valid (similar to json version check). This ensures across the whole repo we can rely on it.